### ISSUE TITLE
State / what needs to be done?

### ISSUE THREAD

**Issue by fingolfin** [gap-infra/actions#14]
What is the state of this now? What still needs to be done?

- set up some kind of recurring trigger
  - perhaps just a simple time base schedule
  - perhaps it can be modified though to only run if either GAP or the distro changed (resp.: if the container we use changed)  -- but I have no idea how that would work (perhaps the workflow updating the docker image could trigger a job here?!)
- better status overview
  - @FriedrichRober and @wucas are working on this, I think?
- ... ??

**Comment by FriedrichRober** [comment]
Yes, Lucas and I are working on better status overview, although finalization depends on our triggers and whether the configuration (docker, test all, etc.) is static or dependent on trigger.

I also do not know how we can trigger jobs from other repositories in gap-infra?

**Comment by fingolfin** [comment]
@FriedrichRober what kind of jobs do you want to trigger and why?

I think they added some capabilities to do this some time ago, maybe [`workflow_call`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call) can do it, or another of the event types listed there. @wilfwilson might also know more ;-)

**Comment by fingolfin** [comment]
See also https://docs.github.com/en/actions/using-workflows/reusing-workflows

**Comment by FriedrichRober** [comment]
I summarized the current state and plans for status overview in the respective draft pull request #17.

I want to call workflows in this repo from other repos. For example I want to run tests if we add a certain comment for a pull request in the gap repository as it is done in [Julia](https://github.com/JuliaLang/julia/pull/43190#issuecomment-976383993) and probably let a bot leave a comment once the tests are finished. But I haven't done anything into this direction yet.

**Comment by wilfwilson** [comment]
Sorry I haven't got round to responding to this. I will respond and create some issues when I have time again.
